### PR TITLE
Ignore dependency tags when replacing version

### PR DESCRIPTION
**Pull request by FrNecas**
Resolves: #730

### CODE REVIEW

File: rebasehelper/tests/plugins/test_spec_hooks.py [hunk 1]
```diff
@@ -83,6 +83,11 @@ def test_replace_old_version_spec_hook(self, spec_object, replace_with_macro):
             expected_result = 'https://test.com/#/1.1/%{name}-hardcoded-version-1.1.0.tar.gz'
         assert test_source[0].split()[1] == expected_result
 
+        # Check if dependency tags are ignored
+        test_recommends = [line for line in new_spec.spec_content.section('%package') if line.startswith('Recommends')]
```
Comment (nforro) [inline rebasehelper/tests/plugins/test_spec_hooks.py]: You could also use `new_spec.get_raw_tag_value('Recommends')`.

File: rebasehelper/plugins/spec_hooks/replace_old_version.py [hunk 2]
```diff
@@ -104,7 +116,7 @@ def run(cls, spec_file: SpecFile, rebase_spec_file: SpecFile, **kwargs: Any):
             for index, line in enumerate(section):
                 # special case Version tag to avoid mistakes when version format changes
                 # e.g. in a rebase from 2.5 to 2.5.1, version would be changed to 2.5.1.1
-                if cls._is_local_source(line) or line.startswith('Version'):
+                if cls._is_local_source(line) or any(line.startswith(tag) for tag in cls.IGNORED_TAGS):
```
Comment (nforro) [inline rebasehelper/plugins/spec_hooks/replace_old_version.py]: The comment mentions only Version.

Comment (FrNecas) [reply rebasehelper/plugins/spec_hooks/replace_old_version.py]: I moved the comment to the list, I think it is clearer this way

File: rebasehelper/tests/plugins/test_spec_hooks.py [hunk 3]
```diff
@@ -83,6 +83,10 @@ def test_replace_old_version_spec_hook(self, spec_object, replace_with_macro):
             expected_result = 'https://test.com/#/1.1/%{name}-hardcoded-version-1.1.0.tar.gz'
         assert test_source[0].split()[1] == expected_result
 
+        # Check if dependency tags are ignored
+        test_recommends = new_spec.get_raw_tag_value('Recommends')
+        assert test_recommends == 'test > 1.0.2'
```
Comment (nforro) [inline rebasehelper/tests/plugins/test_spec_hooks.py]: Just a nitpick, but this could be oneliner.

Comment (FrNecas) [reply rebasehelper/tests/plugins/test_spec_hooks.py]: Good catch

**Pull request merged**